drop-storage/src/lib.rs

```diff
@@ -69,12 +80,33 @@ impl Storage {
 
         match &transfer.files {
             TransferFiles::Incoming(files) => {
+                info!(self._logger, "incoming files len {}", files.len());
+
                 for file in files {
+                    info!(
```

Comment:
I would say let's change it to `debug!()` because we're logging here for example paths, which was not good from the security point of view

Reply:
debug! makes sense here